x/valset-pref/valpref-module/module.go

```diff
@@ -0,0 +1,210 @@
+package validator_preference
```

Comment:
There shouldn't be a `valpref-module` directory/pkg. The `module.go` should exist under the root package, `x/valset-pref` :)

Reply:
hmm is there a reason why we dont follow this for TWAP module?

Reply:
The twap module is the new standard for Osmosis, which follows this folder structure

Reply:
I find this very very strange, but OK :P

Feel free to resolve.